atomics/T1069/T1069.yaml

```diff
@@ -14,9 +14,9 @@ atomic_tests:
   executor:
     name: sh
     command: |
-      dscacheutil -q group
-      dscl . -list /Groups
-      groups
+      if [ -x "$(command -v dscacheutil)" ]; then dscacheutil -q group; else echo "dscacheutil is missing from the machine. skipping..."; fi; fi;
+      if [ -x "$(command -v dscl)" ]; then dscl . -list /Groups; else echo "dscl is missing from the machine. skipping..."; fi;
+      if [ -x "$(command -v groups)" ]; then groups; else echo "groups is missing from the machine. skipping..."; fi;
```

Comment:
Could we add the check for binary existing as actual prereqs as you do in some of the later files in this commit?

Reply:
`dscl` is macOS specific so it is expected not to be present on linux. How do you want to proceed in similar cases?

Reply:
Hadn't thought through this before. we can leave it like this and if we decided to break it out to a macOS atomic and a separate Linux atomic later we can.